backend/Resources/MetadataResource.cs

```diff
@@ -9,16 +10,16 @@ public class MetadataResource
 	{
 		public Guid Uuid { get; set; }
 
-		public string Url { get; set; }
 		public string Description { get; set; }
-		public string FormatName { get; set; }
 
 		public EReleaseState ReleaseState { get; set; }
 
 		public string MetadataTypeName { get; set; }
 
 		public string MunicipalityName { get; set; }
 
+		public IList<DataSourceResource> DataSource { get; set; }
```

Comment:
DataSources?

Reply:
Definetly an error, however this would require significant front-end updates as well. Do we wanna fix it now?

Reply:
No